tests/config_test.go

```diff
@@ -286,4 +282,201 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		})
 
 	})
+
+	Context("With a Secret and a ConfigMap defined", func() {
+
+		Context("With a single volume", func() {
+			var (
+				configMapName string
+				configMapPath string
+				secretName    string
+				secretPath    string
+			)
+
+			BeforeEach(func() {
+				configMapName = "configmap-" + uuid.NewRandom().String()
+				configMapPath = config.GetConfigMapSourcePath(configMapName + "-disk")
+				secretName = "secret-" + uuid.NewRandom().String()
+				secretPath = config.GetSecretSourcePath(secretName + "-disk")
+
+				config_data := map[string]string{
+					"config1": "value1",
+					"config2": "value2",
+					"config3": "value3",
+				}
+
+				secret_data := map[string]string{
+					"user":     "admin",
+					"password": "redhat",
+				}
+
+				tests.CreateConfigMap(configMapName, config_data)
+
+				tests.CreateSecret(secretName, secret_data)
+			})
+
+			AfterEach(func() {
+				tests.DeleteConfigMap(configMapName)
+				tests.DeleteSecret(secretName)
+			})
+
+			It("[test_id:786]Should be that cfgMap and secret fs layout same for the pod and vmi", func() {
+				expectedOutput_cfgMap := "value1value2value3"
+				expectedOutput_Secret := "adminredhat"
+
+				By("Running VMI")
+
+				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdataHighMemory(
+					tests.ContainerDiskFor(
+						tests.ContainerDiskFedora), "#!/bin/bash\necho \"fedora\" | passwd fedora --stdin\n")
+				tests.AddConfigMapDisk(vmi, configMapName)
+				tests.AddSecretDisk(vmi, secretName)
+				tests.RunVMIAndExpectLaunch(vmi, 90)
+
+				By("Checking if ConfigMap has been attached to the pod")
+				vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+				podOutput_cfgMap, err := tests.ExecuteCommandOnPod(
+					virtClient,
+					vmiPod,
+					vmiPod.Spec.Containers[1].Name,
+					[]string{"cat",
+						configMapPath + "/config1",
+						configMapPath + "/config2",
+						configMapPath + "/config3",
+					},
+				)
+				Expect(err).To(BeNil())
+				Expect(podOutput_cfgMap).To(Equal(expectedOutput_cfgMap))
+
+				By("Checking mounted ConfigMap image")
+				expecter, err := tests.LoggedInFedoraExpecter(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				defer expecter.Close()
+
+				_, err = expecter.ExpectBatch([]expect.Batcher{
+					// mount ConfigMap image
+					&expect.BSnd{S: "sudo su -\n"},
+					&expect.BExp{R: "#"},
+					&expect.BSnd{S: "mount /dev/sda /mnt\n"},
+					&expect.BSnd{S: "echo $?\n"},
+					&expect.BExp{R: "0"},
+					&expect.BSnd{S: "cat /mnt/config1 /mnt/config2 /mnt/config3\n"},
+					&expect.BExp{R: expectedOutput_cfgMap},
+				}, 200*time.Second)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
You can add logging for better debug messages. As noted here https://github.com/kubevirt/kubevirt/pull/2412/files#diff-d47bf036797965d742615e64adc3b777R443

The same for other expecters.